fe/fe-core/src/main/java/org/apache/doris/statistics/BaseAnalysisTask.java

```diff
@@ -484,6 +484,11 @@ protected void runQuery(String sql) {
         try (AutoCloseConnectContext a  = StatisticsUtil.buildConnectContext(false)) {
             stmtExecutor = new StmtExecutor(a.connectContext, sql);
             ColStatsData colStatsData = new ColStatsData(stmtExecutor.executeInternalQuery().get(0));
+            if (!colStatsData.isValid()) {
+                String message = String.format("ColStatsData is invalid, skip analyzing. %s", colStatsData.toSQL(true));
+                LOG.info(message);
```

Comment:
warning? could we get table column and invalid data from log message?

Reply:
Fixed. We could get column name, table, db and catalog id in the log message.